mappings/net/minecraft/loot/function/EnchantRandomlyLootFunction.mapping

```diff
@@ -1,14 +1,14 @@
 CLASS net/minecraft/class_109 net/minecraft/loot/function/EnchantRandomlyLootFunction
-	FIELD field_1030 enchantments Ljava/util/List;
+	FIELD field_1030 types Ljava/util/List;
 	FIELD field_1031 LOGGER Lorg/apache/logging/log4j/Logger;
 	METHOD <init> ([Lnet/minecraft/class_4570;Ljava/util/Collection;)V
 		ARG 1 conditions
-		ARG 2 enchantments
+		ARG 2 types
 	METHOD method_488 ([Lnet/minecraft/class_4570;)Lnet/minecraft/class_117;
 		ARG 0 conditions
 	METHOD method_489 builder ()Lnet/minecraft/class_120$class_121;
 	CLASS class_110 Factory
 	CLASS class_4954 Builder
-		FIELD field_23090 enchantments Ljava/util/Set;
+		FIELD field_23090 types Ljava/util/Set;
```

Comment:
I think we should keep calling things like these `enchantments`, even though they're types. Look at biome code, for example, which uses `entities` for lists of `EntityType`s which are allowed in the biome.

Reply:
Yeah, wasn't sure if I should keep those. I'll revert that then. Same for parameters and everything, or just collections as in this case?

Reply:
https://minecraft.gamepedia.com/Loot_table#Functions
the json field is `enchantments`